src/NovelRT/Graphics/Camera.cpp

```diff
@@ -33,7 +57,6 @@ namespace NovelRT::Graphics {
   }
 
   void Camera::forceResize(const Maths::GeoVector2<float>& windowSize) {
-    setProjectionMatrix(Maths::GeoMatrix4x4<float>(glm::ortho<float>(0, windowSize.getX(), windowSize.getY(), 0, 0, 65535)));
-    setViewMatrix(Maths::GeoMatrix4x4<float>(glm::scale(glm::vec3(windowSize.getX() / 1920.0f, windowSize.getY() / 1080.0f, -1.0f))));
+    forceResizeCallback()(this, windowSize);
```

Comment:
Have you confirmed this provides the correct behaviour at runtime?

Reply:
I ran the sample and personally saw no difference. I ran it with perspective and orth camera.
I might be wrong so a double/triple/quadruple check is welcome.

Reply:
All you need to check is resize the window at runtime. Did you do that and did it work?

Reply:
Yeah the resize worked. Only time I had a failure it was on the persepctive camera when there was no check for Y == 0.
Now there's a check and stuff is correctly resizing.

Reply:
Ok awesome.